mlrun/runtimes/sparkjob.py

```diff
@@ -213,7 +215,26 @@ def _get_gpu_type_and_quantity(resources):
         gpu_quantity = resources[gpu_type[0]] if gpu_type else 0
         return gpu_type[0] if gpu_type else None, gpu_quantity
 
+    def _validate(self, runobj: RunObject):
+        # KeyError is used because it is raised and handled correctly and eventually shows
+        # the informative message to the user
+        # TODO - Change to use MLRunError types when fastapi framework handles the internal exceptions correctly
+
+        # validating length limit for sparkjob's function name
+        try:
+            verify_field_regex("name", runobj.metadata.name, sparkjob_name)
```

Comment:
`"name"` is the field name that will be logged to the user explaining him what's wrong, name is too generic and might confuse the user on where is the problem, please change to `"run.metadata.name"`